optuna/study.py

```diff
@@ -322,6 +322,25 @@ def _should_stop() -> bool:
     def set_user_attr(self, key: str, value: Any) -> None:
         """Set a user attribute to the study.
 
+        Example:
+
```

Comment:
It might be kind for users to note that "Please see the example code of the `user_attrs` method." What do you think?

Reply:
Nice idea!

Reply:
@HideakiImamura 
I'm sorry, but I couldn't get the point. If users follow this instruction, they will see the same example as the example in this PR. Do you intend to remove this duplicated PR and guide users to the example in the `user_attrs` section?

Reply:
Sorry. 
I will do this:
- Add detailed explanation of this function, referred to [Tutorial](https://optuna.readthedocs.io/en/latest/tutorial/006_attributes.html).
- Note that "Please see the example code of the `user_attrs` method" into `Study.user_attrs`(#1745).

Reply:
Thanks for the suggestion, @norihitoishida! Actually, the content of the example of `Study.set_user_attr` and `Study.user_attr` are same. Therefore, I initially thought that relating these two examples is reasonable. However, we don't have to explain these two examples are related here. Now, I think there is nothing that we should do here. Sorry, fo the confusion.

Reply:
@HideakiImamura 
Thanks for detail, I got it!